specification.json

```diff
@@ -383,6 +376,13 @@
             "RFC 2119 keyword": "MUST",
             "children": []
         },
+        {
+            "id": "Condition 4.3.8",
+            "machine_id": "condition_4_3_8",
+            "content": "`finally` is a reserved word in the language.",
+            "RFC 2119 keyword": null,
+            "children": []
```

Comment:
I think this should have children? You can make this happen by increasing the heading level of the thing under it.